### ISSUE TITLE
Handling unsupported emojis?

### ISSUE THREAD

**Issue by Tymewalk** [Tymewalk/Tymoji#18]
How should unsupported emojis (like ~money_mouth~ on GitHub theme) be handled?
I was thinking a red box with an X through the middle.

**Comment by herohamp** [comment]
😄

**Comment by Tymewalk** [comment]
It could be coded like this:

Data list:
    
    ...
    ["~supported_emoji~", "emojifilename.png", "~supported_emoji~"],
    ["~unsupported~", "NULL", "~unsupported~ (not supported by this theme)"],
    ...

Rendering code:

    if (emojiData[j][1] == "NULL") {
      // Use the box here
    } else {
      // Do things normally
    }

**Comment by joker314** [comment]
Maybe it should instead fall back onto another theme?

**Comment by Tymewalk** [comment]
@joker314 You mean like by default use Tymoji theme (since that's the one that will always have every emoji)?

**Comment by joker314** [comment]
Yes, that is exactly what I think. (Or maybe we could let them choose what they want to fallback on, and have Tymoji as the fallback of the fallback? I dunno)

**Comment by Tymewalk** [comment]
@joker314 Fallback should include only sets with all supported Tymojis - EmojiOne for example.

If, for ANY REASON, it fails to find it in the fallback, Tymoji is the default.

**Comment by joker314** [comment]
Is this still an active issue?
